cartoframes/viz/kuviz.py

```diff
@@ -0,0 +1,47 @@
+from carto.kuvizs import KuvizManager
+from carto.exceptions import CartoException
+
+
+class Kuviz(object):
+    PRIVACY_PUBLIC = 'public'
+    PRIVACY_PASSWORD = 'password'
+
+    def __init__(self, context, id, url, name, privacy=PRIVACY_PUBLIC):
+        self.cc = context
+        self.id = id
+        self.url = url
+        self.name = name
+        self.privacy = privacy
+
+    @classmethod
+    def create(cls, context, html, name, password=None):
+        carto_kuviz = _create_carto_kuviz(context, html, name, password)
+        _validate_carto_kuviz(carto_kuviz)
+        return cls(context, carto_kuviz.id, carto_kuviz.url, carto_kuviz.name, carto_kuviz.privacy)
+
+    @classmethod
+    def all(cls, context):
+        # km = KuvizManager(context.auth_client)
+        # return km.all()
+        pass
+
+    def update(self, html, name, password=None):
+        pass
+
+    def delete(self):
+        pass
+
+
+def _validate_carto_kuviz(carto_kuviz):
+    if not carto_kuviz or not carto_kuviz.url or not carto_kuviz.id or not carto_kuviz.name:
```

Comment:
Would it make sense having this validation in the carto-python side? Could be just a `kuviz.is_valid()` method.